Cargo.toml

```diff
@@ -16,14 +16,17 @@ byteorder = "1.4.3"
 crc = "3.0.0"
 log = { version = "0.4.17", optional = true }
 env_logger = { version = "0.9.0", optional = true }
+core2 = "0.4"
```

Comment:
This crate has not been updated in 2 years and has a "CI failing" badge, so I'm not sure it's a good choice.

Reply:
I've opened a pull request to fix CI builds in [core2/#21](https://github.com/technocreatives/core2/pull/21).